src/drawer/components/MoreProductSection/MoreProductSection.tsx

```diff
@@ -10,26 +17,34 @@ export const MoreProductSection = ({
   providerName: string;
   providerId: string;
 }) => {
-  const { isSuccess, data } = useGetProductByProvider({ providerId });
+  const { data } = useGetProductByProvider({ providerId });
+
```

Comment:
`data` 대신 `productList` 같이 alias를 사용해서 destructuring을 해주면 가독성이 더 좋아질 것 같아요

```suggestion
  const { data: productList } = useGetProductByProvider({ providerId });
```

Reply:
음 굿굿.
근데 제가 지금 본문에 적어둔 체리 PR에서 뽑아올 커밋이 있어서 거기 픽해오고 수정할게여 이건

Reply:
data에 products(개발자가 제작한 프로덕트)만 담겨오는 게 아니라 이제 providerName도 함께 와서, productList말고 그냥 data에 바로 구조분해 할당 적용시켜두는 게 더 깔끔할 거 같아 그렇게 커밋해뒀습니당~~